src/views/documents/DocumentsPrintingView.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="section documents-view">
+    <html-header :title="getDocumentTypeTitle(documentType)" />
+
+    <div class="documents-container">
+      <div v-for="(doc, index) in promises" :key="index" class="column">
+        <component
+          :is="documentType + '-view'"
+          :document-type="doc.documentType"
+          :version="doc.version"
+          :promise="doc.promise"
+          :document-id="doc.documentId"
+          :expected-lang="doc.expectedLang"
+          :fields="doc.fields"
+        />
```

Comment:
Page break in a middle of a topo is painfull, we can use this: 

Add a div : `<div class="page-break" />`, and CSS : 

```css
<style scoped>
.page-break {
  page-break-after: always;
}
</style>
```

Reply:
I had in mind that it was an improvement to be able to print your topos without wasting every time half a page... but if you think it is better to add a page break I'll include your suggestion.

The best (for the future?) would be that pressing the print button open a piece of UI where user can choose different printing options (keeping image or not, page break at the end of a topo...)

Reply:
> The best (for the future?) would be that pressing the print button open a piece of UI where user can choose different printing options (keeping image or not, page break at the end of a topo...)

Indeed. if you have the motivation, it can be the very first option. Actually, I feel that the auto-open of the print page when documents are loaded is a little bit weird. it could be a good way to go: 

* A `div` at the top of the page, with an parameters' set design. It includes : 
  * a tick box with a label `start each document on top of each page`
  * a disabled button with `loading...`
  * when all documents are loaded, the  `loading...` becomes `print`

WDYT ?